app/views/health/qualifying_activities/update.js.coffee

```diff
@@ -0,0 +1,4 @@
+html = "<%= j render('health/sdh_case_management_notes/activities', qa: @qa) %>"
+$('.modal:visible .close').trigger('click')
+$container = $('.jActivityList')
+$container.replaceWith(html)
```

Comment:
replace with `<%= j render 'create' %>` (but we need to test)  the JS code is the same.